.changeset/brave-pots-act.md

```diff
@@ -0,0 +1,5 @@
+---
+"@guardian/cdk": major
```

Comment:
Is this really a breaking change? Or is it a minor?

Reply:
It's breaking because the previous default was to use X86 rather than ARM

Reply:
Ah! Hadn't spotted that. Are there any scenarios where we'd ever use x86? I wonder if a "paved road" solution would be to not expose `cpuArchitecture` at all. Where a service not be be ARM compatible, it'll use an [escape hatch](https://docs.aws.amazon.com/cdk/v2/guide/cfn_layer.html) to change the architecture.

Reply:
Sorry I missed this! Sounds good, have updated